helm/fluent-logshipping-app/templates/priorityClass.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.fluentbit.customEnabled }}
+apiVersion: scheduling.k8s.io/v1
+description: High-priority class for fluentbit pods only.
+kind: PriorityClass
+metadata:
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: {{ .Values.fluentbit.priorityClass.name }}
+preemptionPolicy: PreemptLowerPriority
+value: 200000000
```

Comment:
I think it's too high :D

Reply:
Oh no it's fine :)

Reply:
Well it's still lower than prometheus which already has the lowest priority among those deployed :shrug:

Reply:
Yes I was wrong with the number of zeros :D

Reply:
I thought Prometheus was 500_000 not 500_000_000

Reply:
Oh ok x)